packages/devtools/src/FlattenView.tsx

```diff
@@ -92,9 +97,10 @@ export const FlattenView: React.FC<FlattenViewProps> = props => {
       </div>
       <div className={classes.rules}>
         <ViewContext.Provider value={contextValue}>
-          {filteredSlots.map(({ slot, rules }) => (
-            <SlotCSSRules key={slot} slot={slot} atomicRules={rules} />
-          ))}
+          {filteredSlots.map(({ slot, rules }) => {
+            const key = hash(slot + rules.map(rule => rule.cssRule).join(''));
```

Comment:
Let's not use indexes as it can strike unpredictably later.

```suggestion
            const key =slot + rules.map(rule => rule.cssRule).join('');
```

I would remove hashing as it's not going to DOM anyway.